Sources/ProjectSpec/Breakpoint.swift

```diff
@@ -166,8 +183,17 @@ extension Breakpoint: JSONObjectConvertible {
             } else {
                 scope = .objectiveC
             }
+            if jsonDictionary["stopOnStyle"] != nil {
+                let stopOnStyleString: String = try jsonDictionary.json(atKeyPath: "stopOnStyle")
+                if let stopOnStyle = StopOnStyle(stopOnStyleString) {
+                    self.stopOnStyle = stopOnStyle
+                } else {
+                    throw SpecParsingError.unknownBreakpointStopOnStyle(stopOnStyleString)
+                }
+            } else {
+                stopOnStyle = .throw
```

Comment:
This is a change in behaviour. It used to be optional, but now has `.throw` as default. If that truly is the default, we should make it a non optional. Maybe keeping default as nil is better though?

Reply:
@yonaskolb 
> If that truly is the default, we should make it a non optional.

Could you tell me the reason?

#### Here is my thought:
Basuse `scrop` and `stopOnStyle` are only required when `type == .exception`, and has a default value, I should do something like below for `scrop` and `stopOnStyle` when `type == .exception`.

``` swift
enabled = jsonDictionary.json(atKeyPath: "enabled") ?? true
```